src/Uno.UI/UI/Xaml/Controls/Primitives/SelectorItem.cs

```diff
@@ -65,7 +66,7 @@ private enum ManipulationUpdateKind
 
 		private string _currentState;
 		private uint _goToStateRequest;
-		private DateTime _pauseStateUpdateUntil;
+		private long _pauseStateUpdateUntil;
```

Comment:
Let's use a TimeSpan instead, it makes it easier to read, and there's no performance penalty.

Reply:
Does it need to be absolute? Do we expose that value anywhere?

Reply:
No, could be relative to "something", but we do need to change the logic then and the name of the field :)